projects/manager/src/app/datasets/add-dataset/add-dataset.component.html

```diff
@@ -0,0 +1,82 @@
+<h2 mat-dialog-title>Create new dataset</h2>
```

Comment:
Why is this modal? Other stuff is not modal, here.

Moreover, if users edit rather complex loading infos and click to the side, it just closes. Maybe this is not right for a component, where you do complex stuff?

Reply:
it's modal because you are not supposed to use anything else in the app while creating the data. I mean we can also put it where the dataset-editor is, but I think that's more confusing for the user. We could replace the whole dataset-view with listing and editor with the new dialog, but that's more complicated and also overkill I think.

We could have a confirmation dialog before closing the add dataset dialog though.

Reply:
Ok, but then prevent accidentally leaving the filled dialog.

Reply:
c7516daf8afa2029792d65c321c3e0388f913c7b

Reply:
Does not work when you insert something into the loading info field.

Reply:
3d1fd27de8dac39946400104d4057f4e886cf524